bot.py

```diff
@@ -554,7 +553,7 @@ def get_repo_info(self, channel_id):
             org=org,
             repo=repo,
         )
-        unchecked = await self.translate_slack_usernames(unchecked_authors)
+        unchecked = set(await self.translate_slack_usernames(unchecked_authors))
```

Comment:
`set` here might be redundant here but doesn't hurt either

Reply:
Ah good catch